ci/scripts/utils/launch_java_agent.sh

```diff
@@ -74,7 +74,7 @@ host=$(hostname)
 
 source "${HOMEGFS_}/ush/detect_machine.sh"
 case ${MACHINE_ID} in
-  hera | orion | hercules | wcoss2 | gaea)
+  hera | orion | hercules | wcoss2 | gaea | gaea6 )
```

Comment:
`gaea` is deprecated in favor of `gaeac6` and `gaeac5`.  
`gaea6` should be `gaeac6`
```suggestion
  hera | orion | hercules | wcoss2 | gaeac6 )
```

Reply:
I saw that already when I was working on this and had made the update